dlp-terraform/ecs/ecr.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_ecr_repository" "training" {
```

Comment:
should we also ensure that this ECR repo exists in us-east-1 region?

Reply:
Everything is in us-east-1 since the region is set in main.tf

Reply:
got it!